lib/foreman/gettext/debug.rb

```diff
@@ -1,43 +1,47 @@
 require 'fast_gettext'
 
 # include this module to see translations in the UI
-module Foreman::Gettext::Debug
-  DL = "\u00BB".encode("UTF-8") rescue '>'
-  DR = "\u00AB".encode("UTF-8") rescue '<'
+module Foreman
+  module Gettext
+    module Debug
```

Comment:
Just a note: this is only needed with the classic loader, when we switch to zeitwerk, this shouldn't matter :)